Makefile

```diff
@@ -106,6 +106,16 @@ deps:
 	test -f $(BINPATH)/golangci-lint || curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(BINPATH) v1.21.0
 	test -f $(BINPATH)/shellcheck || curl -sfL "https://storage.googleapis.com/shellcheck/shellcheck-v0.4.7.linux.x86_64.tar.xz" | tar -xJv --strip-components=1 -C $(BINPATH)
 
+.PHONY: pre-commit-install
+pre-commit-install:
+	test -f $(BINPATH)/bin/pre-commit || curl -sfL https://pre-commit.com/install-local.py | HOME=$(BINPATH) python -
+	$(BINPATH)/bin/pre-commit install -t pre-commit -t commit-msg
+
+.PHONY: pre-commit-uninstall
+pre-commit-uninstall:
+	test -f $(BINPATH)/bin/pre-commit || curl -sfL https://pre-commit.com/install-local.py | HOME=$(BINPATH) python -
```

Comment:
if pre-commit is not installed and we are calling uninstall, why install and uninstall it? we should just exit directly

Reply:
The purpose is to remove the .git/hook/pre-commit which is installed by pre-commit, maybe the better way is to delete the file `.git/hook/pre-commit` directly.